pytuya/__init__.py

```diff
@@ -319,3 +330,100 @@ def set_timer(self, num_secs):
         log.debug('set_timer received data=%r', data)
         return data
 
+class BulbDevice(XenonDevice):
```

Comment:
Consider sub-classing the Outlet type. One could move things from outlet up, or even add another intermediate class to the model but I think this is reasonable and leaves the option to do other things in the future.

This would also likely provide timer support too.